consumer_test.go

```diff
@@ -298,6 +350,8 @@ func TestConsumerInterleavedClose(t *testing.T) {
 }
 
 func TestConsumerBounceWithReferenceOpen(t *testing.T) {
+	t.Skip("This is not yet working due to concurrency on the mock broker")
```

Comment:
You'll have to add four offset responses (newest/oldest for two partitions) but I didn't think this test was particularly fragile. It should be ok as long as you don't return the second offset pair until after the first `ConsumePartition` call has returned.